main.py

```diff
@@ -1,10 +1,46 @@
+import json
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    file_name = "players.json"
+    with open(file_name) as file:
+        data = json.load(file)
+
+    for player_name, player_info in data.items():
+
```

Comment:
In my opinion, we don't need to add an empty line from which we start writing something in our code block)